src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -234,7 +231,7 @@ private static function make(self $container, string $id, int $invalidBehavior):
         } catch (\Exception $e) {
             unset($container->services[$id]);
 
-            throw $e;
```

Comment:
if the exception is already a PSR one, should we rethrow it as is instead of wrapping it ?